lib/hiera/config.rb

```diff
@@ -10,8 +10,8 @@ class << self
     # @return [Hash] representing the configuration.  e.g.
     #   {:backends => "yaml", :hierarchy => "common"}
     def load(source)
-      @config = {:backends => "yaml",
-                 :hierarchy => "common",
+      @config = {:backends => ["yaml"],
+                 :hierarchy => ["nodes/%{::trusted.certname}", "common"],
```

Comment:
does Hiera have any tests for this sort of thing, e.g. what is the state of `config` after `load` is called?